rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -91,28 +86,23 @@ class BridgeSupportReleaseBtcTest {
     @BeforeEach
     void setUpOnEachTest() {
         signatureCache = new BlockTxSignatureCache(new ReceivedTxSignatureCache());
-        bridgeConstants = new BridgeRegTestConstants();
-        federationConstants = bridgeConstants.getFederationConstants();
-        networkParameters = bridgeConstants.getBtcParams();
-        activationsBeforeForks = ActivationConfigsForTest.genesis().forBlock(0);
-        activeFederation = getFederation();
+        activeFederation = P2shErpFederationBuilder.builder().build();
         repository = spy(createRepository());
         eventLogger = mock(BridgeEventLogger.class);
-        utxo = buildUTXO();
-        provider = initProvider(repository, activationMock);
-        federationStorageProvider = initFederationStorageProvider(repository, activationMock);
+        provider = initProvider();
+        federationStorageProvider = initFederationStorageProvider();
         bridgeSupportBuilder = BridgeSupportBuilder.builder();
         feePerKbSupport = mock(FeePerKbSupportImpl.class);
-        when(feePerKbSupport.getFeePerKb()).thenReturn(bridgeConstants.getFeePerKbConstants().getGenesisFeePerKb());
-        bridgeSupport = spy(initBridgeSupport(eventLogger, activationMock));
+        when(feePerKbSupport.getFeePerKb()).thenReturn(Coin.valueOf(5_000L));
```

Comment:
Why `5_000` for the fee per kb? I executed the getFeePerKb method and got the result of `24000`. I think the  closer the value is to mainnet value, the better. Wdyt?

Reply:
Just a reasonable value, it's 24,000 in mainnet now but that can change in any minute